taskcluster/ci/config.yml

```diff
@@ -2,7 +2,11 @@
 trust-domain: mobile
 treeherder:
     group-names:
+        'forPerformanceTest': 'Builds made for Raptor and other performance tests'
         'I': 'Docker Image Builds'
+        'nightly': 'Nightly-related tasks'
+        'nightly-legacy': 'Nightly-related tasks that ship to https://play.google.com/store/apps/details?id=org.mozilla.fenix'
```

Comment:
Is it possible to use either `camelCase` or `kebab-case` for all the group names here?

Reply:
Good call. I changed it for `nightlyLegacy` because `forPerformanceTest` is already used.